packages/dds/test-dds-utils/src/index.ts

```diff
@@ -19,3 +19,4 @@ export {
 	Synchronize,
 	replayTest,
 } from "./ddsFuzzHarness";
+export { MinimizationTransform } from "./minification";
```

Comment:
Note for reviewers: this is transitively referenced by an existing export and therefor needs to be exported.